tribe-ext-limit-week-view-time-range.php

```diff
@@ -0,0 +1,315 @@
+			$priority = 5;
+			// Plugin in which the file was loaded from = 20
+			// Events Pro = 25
+			// Tickets = 17
+
+			// Which folder in your plugin the customizations will be loaded from.
+			$custom_folder[] = 'tribe-customizations';
+
+			// Builds the correct file path to look for.
+			$plugin_path = array_merge( (array) trailingslashit( plugin_dir_path( __FILE__ ) ),
+			                            (array) $custom_folder,
+			                            array_diff( $template->get_template_folder(), [ 'src', 'views' ] ) );
+
+			/*
+			 * Custom loading location for overwriting file loading.
+			 */
+			$folders[ $plugin_name ] = [
+				'id' => $plugin_name,
+				'namespace' => $plugin_name, // Only set this if you want to overwrite theme namespacing
+				'priority' => $priority,
+				'path' => $plugin_path,
+			];
+
+			return $folders;
+		}
+
+		/**
+		 * Filters the hours of week view.
+		 * Needed for V1.
+		 *
+		 * @param array $hours
+		 *
+		 * @return array
+		 */
+		public function filter_week_hours( array $hours ) {
+			$options = $this->get_all_options();
+
+			// Set the desired times here, pulls from settings
+			$start_of_day = $options['start_time'];
+			$end_of_day   = $options['end_time'];
+
+			// Fallback
+			if ( ! is_numeric( $start_of_day ) || $start_of_day < 0 || $start_of_day > 23 ) {
+				$start_of_day = 0;
+			}
+			if ( ! is_numeric( $end_of_day ) || $end_of_day < 1 || $end_of_day > 23 || $end_of_day < $start_of_day ) {
+				$end_of_day = 24;
+			}
+
+			foreach ( $hours as $hour => $formatted_hour ) {
+				if ( $hour < $start_of_day || $hour >= $end_of_day ) {
+					unset( $hours[ $hour ] );
+				}
+			}
+
+			return $hours;
+		}
+
+		/**
+		 * Check if we have a sufficient version of PHP. Admin notice if we don't and user should see it.
+		 *
+		 * @link https://theeventscalendar.com/knowledgebase/php-version-requirement-changes/ All extensions require PHP 5.6+.
+		 *
+		 * @return bool
+		 */
+		private function php_version_check() {
+			$php_required_version = '7.0';
+
+			if ( version_compare( PHP_VERSION, $php_required_version, '<' ) ) {
+				if ( is_admin() && current_user_can( 'activate_plugins' ) ) {
+					$message = '<p>';
+					$message .= sprintf( __( '%s requires PHP version %s or newer to work. Please contact your website host and inquire about updating PHP.',
+					                         'tribe-ext-limit-week-view-time-range' ),
+					                     $this->get_name(),
+					                     $php_required_version );
+					$message .= sprintf( ' <a href="%1$s">%1$s</a>', 'https://wordpress.org/about/requirements/' );
+					$message .= '</p>';
+					tribe_notice( 'tribe-ext-limit-week-view-time-range-php-version', $message, [ 'type' => 'error' ] );
+				}
+
+				return false;
+			}
+
+			return true;
+		}
+
+		/**
+		 * Check if we have the required TEC view. Admin notice if we don't and user should see it.
+		 *
+		 * @return bool
+		 * @example
+		 *
```

Comment:
- empty `@example`?
- Remove blank line

Reply:
Removed method as not needed.